pkgs/by-name/gr/graphicsmagick/package.nix

```diff
@@ -47,7 +48,7 @@ stdenv.mkDerivation (finalAttrs: {
     libwebp
     libxml2
     zlib
-  ];
+  ] ++ lib.optional libheifSupport libheif;
```

Comment:
I'd use lib.optionals here,
simply because it's more "safe".

Reply:
You are right, it has been changed.